components/automate-ui/src/app/modules/infra-proxy/cookbooks/cookbooks.component.html

```diff
@@ -1,6 +1,14 @@
 <section class="cookbooks">
   <chef-loading-spinner *ngIf="cookbooksListLoading" size="50"></chef-loading-spinner>
-  <ng-container  *ngIf="!cookbooksListLoading">
+  <chef-toolbar *ngIf="authFailure">
+      <div class="empty-state">
+        <p>Unable to show cookbooks because admin key is invalid.</p>
+      </div>
+      <div class="empty-state">
+        <chef-button primary (click)="ResetKeyTabRedirection()">Reset Admin Key</chef-button>
+      </div>
```

Comment:
Wondering why two separate classes? Is there a reason this would not work?
```suggestion
      <div class="empty-state">
        <p>Unable to show cookbooks because admin key is invalid.</p>
        <chef-button primary (click)="ResetKeyTabRedirection()">Reset Admin Key</chef-button>
      </div>
```

Reply:
Was playing around with UI forgot to remove it, Done now Thanks :+1: